src/handle-click.ts

```diff
@@ -40,8 +40,8 @@ export const handleClick = (
           entityId: actionConfig.entity
             ? actionConfig.entity
             : config.entity
-            ? config.entity
-            : config.camera_image
+              ? config.entity
```

Comment:
accidental?

Reply:
My formatter does it this way, it's just indentation, or am I missing smth?

Reply:
Well I don't want it going back and forth depending on which formatter someone is using. But that just means I need to add some linter configs 😄